qa/suites/fs/upgrade/upgraded_client/tasks/2-workload/stress_tests/upgraded_client_kernel_untar.yaml

```diff
@@ -0,0 +1,15 @@
+tasks:
+- install:
+    extra_system_packages:
+      - bison
+      - flex
+      - elfutils-libelf-devel
+      - openssl-devel
+      - NetworkManager
+      - iproute
+      - util-linux
+- workunit:
+    clients:
+      all:
+        - kernel_untar_build.sh
+
```

Comment:
Maybe you could add this file to `.qa/suites/fs/workload/tasks/5-workunit/suites/` and then link it here just likes the `iozone.yaml`?

The others looks nice to me.

Reply:
There already exists `kernel_untar_build.yaml` in `.qa/suites/fs/workload/tasks/5-workunit`, but it doesn't install the packages that this test needs, it plain simple `kernel_untar_build` test. So now if i move this to `/tasks/5-workunit`, i hope that is fine as there could be suite(s) where the preparation yamls take care of the installation of these packages and this would then unnecessarily try install them again.

Reply:
> There already exists `kernel_untar_build.yaml` in `.qa/suites/fs/workload/tasks/5-workunit`, but it doesn't install the packages that this test needs, it plain simple `kernel_untar_build` test. So now if i move this to `/tasks/5-workunit`, i hope that is fine as there could be suite(s) where the preparation yamls take care of the installation of these packages and this would then unnecessarily try install them again.

You should fix it anyway in `.qa/suites/fs/workload/tasks/5-workunit`. Just make it to work in all the cases, like the one of this PR hit. IMO it's no harm to always install the depending packages.

Reply:
Alright, I'll make the change

Reply:
Done, thanks.